tests/mpi/constraints_consistent_02.mpirun=3.output

```diff
@@ -0,0 +1,17 @@
+Working on rank 0 of 3
```

Comment:
Is the output already generated with https://github.com/dealii/dealii/pull/14905?

Reply:
Yes, this uses the code of #14905.

Reply:
Are you sure that the output is up to date? Where is `Working on rank` coming from and why do not all processes output `Output 0`, `Output 1`, and `Output 2`?

Reply:
The output should be:
```

DEAL:0::Output 0
DEAL:0::Output 1
    0 1:  0.500000
    0 3:  0.500000
    0 2:  0.500000
    2 3:  1.50000
DEAL:0::Output 2
    0 1:  0.500000
    0 3:  1.25000
    2 3:  1.50000

DEAL:1::Output 0
    0 1:  0.500000
    0 2:  0.500000
    2 3:  1.50000
DEAL:1::Output 1
    0 1:  0.500000
    0 3:  0.500000
    0 2:  0.500000
    2 3:  1.50000
DEAL:1::Output 2
    0 1:  0.500000
    0 3:  1.25000
    2 3:  1.50000


DEAL:2::Output 0
    0 1:  0.500000
    0 3:  0.500000
DEAL:2::Output 1
    0 1:  0.500000
    0 3:  0.500000
    0 2:  0.500000
    2 3:  1.50000
DEAL:2::Output 2
    0 1:  0.500000
    0 3:  1.25000
    2 3:  1.50000

```